### DISCUSSION TITLE
electron+typescript build err

### DISCUSSION THREAD

**Discussion by orangebom** [quasarframework/quasar#14820] [General - Components / Directives / etc]
### Description

Project Build Options：
![53e7b908cc30a911f0cc1ffd66dd19b](https://user-images.githubusercontent.com/34227259/200105642-c59d7a18-1ae0-431d-8048-eb47cc4332a0.png)
 after run command：
```
quasar build -m electron
```
report error:
```
App • ⚠️   FAIL  electron-packager could not build

Error: ENOTEMPTY: directory not empty, rmdir 'C:\Users\ORANGE~1\AppData\Local\Temp\electron-packager\win32-x64\Quasar App-win32-x64'

node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[  UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block,   or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "undefined".    
] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

env：
```
"quasar": "^2.6.0",
"vue": "^3.0.0",
"electron": "^21.2.2",
"eslint": "^8.10.0",
"@quasar/app-webpack": "^3.0.0",
"@types/node": "^12.20.21",
```
pc:
Windows 10 19043.2130
ps:
It is normal to create using javascript and can generate an exe

### Areas

TypeScript Support

**Comment by pdanpdan** [comment]
What is the path where you have your code?
It looks like you have some files in `C:\Users\ORANGE~1\AppData\Local\Temp\electron-packager\win32-x64\Quasar App-win32-x64` that are hidden or protected or created by another user.

**Reply by orangebom** [reply]
Code in the D:
Opening this folder doesn't show anything special, nor does running it with administrator privileges

**Reply by orangebom** [reply]
I am looking for other computer creation projects, there is a problem of agreement, but there is no file error reported. The error is as follows:
![d4356c9f5b7c6262a089ae9a1974076](https://user-images.githubusercontent.com/34227259/200106556-12061a28-c4d3-43f5-814d-180330ae5752.png)

**Reply by orangebom** [reply]
That's it. It turns out the proxy is set，thank

**Comment by huagaojian** [comment]
I had the same problem as you. Most probably there is a network error.
When I configured the npm mirror, the compilation passed.

Settings below:
```
yarn config set registry https://registry.npm.taobao.org
yarn config set disturl https://npm.taobao.org/dist
yarn config set electron_mirror https://npm.taobao.org/mirrors/electron/
```
